README.md

```diff
@@ -1,45 +1,54 @@
-# ThermImageAnalysis
-* `CThermal.py` contains classes for sensor value-->temperature conversion from FLIR files (SEQ, images), and the analysis tools for the same, and for splitting SEQ videos
-* `thermal_analysys.py` contains the main code to be run for the analysis tools on the FLIR files.
-* `split_seq.py` splits an SEQ file into multiple FFF files
+# Thermal Image Analysis
+A tool for analyzing and annotating thermal images.
+ - `main.py` contains code for GUI and canvas tools.
+ - `utils.py` contains code for auxiliary functions.
 
+[![Quality check](https://github.com/detecttechnologies/Thermal_Image_Analysis/actions/workflows/qualitycheck.yml/badge.svg)](https://github.com/detecttechnologies/Thermal_Image_Analysis/actions)
 
-## Usage
-* First, install the requirements for using the code in this repo using `pip install -r requirements.txt`
-* For Linux users, check if exiftool is installed, by running ```exiftool``` in the command line.
-* For Windows users, do the following: 
-    * Download exiftool (for your OS) from [here](https://exiftool.org/)
-    * Extract 'exiftool(-k).exe' to the folder where you're running the code from
-    * Rename it to 'exiftool.exe'
-* To run thermal analysis on a FLIR image (FFF/jpg/rjpg/...), run the following command in a terminal:
-    ```bash
-    python thermal_analysis.py <file name>
-    ```  
-  The usage instructions and other notes for this are in the [wiki page](https://github.com/detecttechnologies/Thermal_Image_Analysis/wiki/Instructions-Manual-for-the-Thermal-Image-Analysis-software)
-* To extract FFF(RJPG header that can be used by `thermal_analysis.py` of this repo) and JPG files from an SEQ video, you can run the following command:
-    ```bash
-    python split_seq.py <SEQ file name>
-    ```
+## Features
 
+<p align="center"><img src="assets/images/main.png" /><br/>Main menu</p>
```

Comment:
Not sure how easy this would be, but can you make a compressed gif of your mouse hovering, and the "x", "y" and "Temp" values changing?